lib/config/typescript-prettier.js

```diff
@@ -1,15 +1,10 @@
+const merge = require('merge');
+
 module.exports = {
   extends: ['plugin:shopify/prettier'],
 
-  rules: {
+  rules: merge(require('./rules/typescript-prettier'), {
```

Comment:
I refactored the typescript variant at the same time to import the config. maybe not necessary, but should be no harm.